packages/collector/test/tracing/messaging/node-rdkafka/producer.js

```diff
@@ -22,19 +22,21 @@ const log = require('@instana/core/test/test_util/log').getLogger(logPrefix);
 const express = require('express');
 const port = require('../../../test_util/app-port')();
 const enableDeliveryCb = process.env.RDKAFKA_PRODUCER_DELIVERY_CB === 'true';
-
+const isStream = process.env.RDKAFKA_PRODUCER_AS_STREAM === 'true';
```

Comment:
The producer was auto starting a standard and a stream instance. There was no need to. The tests are bootstrapping the version they need. This improves the speed